sources/osgViewer/eventProxy/StandardMouseKeyboard.js

```diff
@@ -220,6 +220,7 @@ StandardMouseKeyboard.prototype = {
         }
 
         this.getManipulatorController().setEventProxy(this);
+        this.addEventListeners();
```

Comment:
I am not sure you want to call addEventlisteners at each update call. It's called once per frame. You should probably check if it's set or not